lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -154,6 +163,11 @@
     name: notepadplusplus
     version: '6.6'
 
+- name: Install notepadplusplus 32 bit version
+  win_chocolatey:
+    name: notepadplusplus
+    architecture: 'x86'
```

Comment:
For the documentation and examples we tend to not quote where quotes are not needed in YAML. The main reason is that this helps users understand better when quotes are really required in YAML. And a better understanding of how YAML works helps in the long run avoid (common) mistakes.

(It is not consistently done everywhere though...)

Reply:
Thanks @dagwieers - TBH, I could probably do with a better understanding of when quotes are required too! I've learned something myself today :)